model_api/cpp/cmake/model_apiConfig.cmake

```
@@ -1,5 +1,5 @@
 include(CMakeFindDependencyMacro)
-find_dependency(OpenCV)
-find_dependency(OpenVINO)
+find_dependency(OpenCV REQUIRED COMPONENTS core imgproc)
+find_dependency(OpenVINO REQUIRED COMPONENTS Runtime)
```

```suggestion
find_dependency(OpenVINO COMPONENTS Runtime)
```
BTW, `find_dependency` does not require `REQUIRED`, because it inherits `QUIET` or `REQUIRED` from main package.